en/getting_started/flight_modes.md

```diff
@@ -150,32 +166,42 @@ TBD
 
 ### Manual Mode {#manual_fw}
 
+[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/difficulty_4.svg" title="Medium-hard flight mode" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;
```

Comment:
This should be `difficulty_6`. It's more difficult than acro, since there is no stabilization at all.

Reply:
Thanks. I'll reorder so this is 6, acro 5, rattitude 4. stabliized 3, altitude 2, position 1.